tests/testthat/test_table.R

```diff
@@ -15,6 +15,17 @@ test_that("data.frame can be write to and read from csv consistently", {
   expect_equal(b, df2$b)
 })
 
+test_that("empty data.frame can be write to and read from csv consistently", {
```

Comment:
'can be written to'

Reply:
thanks Kenny! I copied this so there is more to fix.